zerver/views/user_settings.py

```diff
@@ -243,7 +243,7 @@ def json_change_notify_settings(
         json_validator=check_bool, default=None
     ),
     wildcard_mentions_notify: Optional[bool] = REQ(json_validator=check_bool, default=None),
-    notification_sound: Optional[str] = REQ(json_validator=check_string, default=None),
+    notification_sound: Optional[str] = REQ(default=None),
```

Comment:
Where is the corresponding JS change for this line?

Reply:
I may have missed the frontend change for this endpoint. Will find and update.